interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -1414,7 +1432,7 @@ public void customMetadata() throws Exception {
 
   @Test(timeout = 10000)
   public void censusContextsPropagated() {
-    if (!serverInProcess()) {
+    if (server == null) {
```

Comment:
This should be `Assume.assumeTrue(server != null)`.  That way it will be clear from the test results that this was skipped rather than passing.

Reply:
Done.